src/v2/components/Layout/Layout.tsx

```diff
@@ -1,125 +1,98 @@
 import { cn } from "@/lib/utils"
 import { Slot } from "@radix-ui/react-slot"
-import clsx from "clsx"
-import { PropsWithChildren, ReactNode } from "react"
-import { forwardRef } from "react"
+import { HTMLAttributes, forwardRef } from "react"
 
-/* -------------------------------------------------------------------------------------------------
- * Layout
- * -----------------------------------------------------------------------------------------------*/
-
-interface LayoutProps extends React.HTMLAttributes<HTMLDivElement> {}
-
-const Layout = ({ children, ...props }: LayoutProps) => (
+const Layout = ({ children, ...props }: HTMLAttributes<HTMLDivElement>) => (
   <div className="flex min-h-screen flex-col" {...props}>
     {children}
   </div>
 )
 
-/* -------------------------------------------------------------------------------------------------
- * LayoutContainer
- * -----------------------------------------------------------------------------------------------*/
-
-interface LayoutContainerProps extends React.HTMLAttributes<HTMLDivElement> {
+interface LayoutContainerProps extends HTMLAttributes<HTMLDivElement> {
   asChild?: boolean
 }
 
 const LayoutContainer = forwardRef<HTMLDivElement, LayoutContainerProps>(
-  ({ children, className, asChild = false }, ref) => {
+  ({ children, className, asChild = false, ...props }, ref) => {
     const Comp = asChild ? Slot : "div"
     return (
       <Comp
-        className={clsx("mx-auto max-w-screen-lg px-4 sm:px-8 md:px-10", className)}
+        className={cn(
+          "mx-auto w-full max-w-screen-lg px-4 sm:px-8 md:px-10",
+          className
+        )}
         ref={ref}
+        {...props}
       >
         {children}
       </Comp>
     )
   }
 )
 
-/* -------------------------------------------------------------------------------------------------
- * LayoutHero
- * -----------------------------------------------------------------------------------------------*/
-
-const LayoutHero = ({ children }: PropsWithChildren) => (
-  <div className="relative">{children}</div>
+const LayoutHero = ({
+  className,
+  children,
+  ...props
+}: HTMLAttributes<HTMLDivElement>) => (
+  <div className={cn("relative pb-40", className)} {...props}>
+    {children}
+  </div>
 )
 
-/* -------------------------------------------------------------------------------------------------
- * LayoutHeadline
- * -----------------------------------------------------------------------------------------------*/
-
-interface LayoutHeadlineProps {
-  title: ReactNode
-}
-
-const LayoutHeadline = ({ title }: LayoutHeadlineProps) => (
-  <LayoutContainer>
-    <h1 className="pt-9 pb-14 font-bold font-display text-4xl text-white tracking-tight sm:text-5xl">
-      {title}
-    </h1>
-  </LayoutContainer>
+const LayoutBanner = ({ className, ...props }: HTMLAttributes<HTMLDivElement>) => (
+  <div
+    className={cn("-z-10 absolute inset-0 overflow-hidden bg-banner", className)}
```

Comment:
The banner is meant to be an element defining the area of its background, since not every page requires a `bg-banner`. Should we overwrite the value on the fly?

Reply:
Yup, that was my idea. Providing a good looking default value here which we can overwrite when needed.